iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/utils/CheckUtils.java

```diff
@@ -141,4 +142,12 @@ public static StringLiteral parameterName(FunctionCall functionCall) {
     }
     return null;
   }
+
+  private static Expression maybeUnwrap(Expression expr) {
```

Comment:
I am not a fan of the words `unwrap` and `maybe` in function names. My suggestion would be `retrieveIdentifierOrExpression`. WDYT?